server/lib/models/import.go

```diff
@@ -0,0 +1,32 @@
+package models
+
+import (
+	"log"
+	"path/filepath"
+	"os"
+)
+
+func create_channels(path string) {
+	
+}
+
+func Import(path string) error {
```

Comment:
__[gometalinter]__
* [golint] exported function Import should have comment or be unexported